packages/radix-ui-themes/src/helpers/input-attributes.ts

```diff
@@ -0,0 +1,51 @@
+// https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attributes
+
+type InputAttributes =
+  | 'accept'
+  | 'alt'
+  | 'autocapitalize'
+  | 'autocomplete'
+  | 'capture'
+  | 'checked'
+  | 'defaultChecked'
+  | 'defaultValue'
+  | 'dirname'
+  | 'disabled'
+  | 'form'
+  | 'formaction'
+  | 'formenctype'
+  | 'formmethod'
+  | 'formnovalidate'
+  | 'formtarget'
+  | 'height'
+  | 'list'
+  | 'max'
+  | 'maxlength'
+  | 'min'
+  | 'minlength'
+  | 'multiple'
+  | 'name'
+  | 'pattern'
+  | 'placeholder'
+  | 'popovertarget'
+  | 'popovertargetaction'
+  | 'readonly'
+  | 'required'
+  | 'size'
+  | 'src'
+  | 'step'
+  | 'type'
+  | 'value'
+  | 'width';
+
+type InputCommonAttributes = 'disabled' | 'form' | 'name' | 'required' | 'defaultValue' | 'value';
+type InputRadioAttributes = InputCommonAttributes | 'defaultChecked' | 'checked';
+
+type NotInputRadioAttributes = Exclude<InputAttributes, InputRadioAttributes>;
+
+export type {
+  InputAttributes,
+  InputCommonAttributes,
+  InputRadioAttributes,
+  NotInputRadioAttributes,
+};
```

Comment:
Entertaining the idea of removing some of the useless attributes/values from the text field input, like `type="file"` or `accept`

Reply:
Let's chat about all this in review

Reply:
Agreed to keep it and remove nonsensical attributes from other input types

Reply:
A funny bit – `<div defaultChecked />` and `<div defaultValue />` is a thing in React